transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodActivity.java

```diff
@@ -177,6 +181,15 @@ public void showConfirmationDialog(@NonNull TransferMethod transferMethod) {
         }
     }
 
+    @Override
+    public void invokeTransferMethodEdit(@NonNull TransferMethod transferMethod) {
```

Comment:
lets call this `invokeTransferMethodUpdate` since we are using `Update` for naming in code

Reply:
fixed

Reply:
updated